### PR TITLE
Code cleanup and features

### PR DESCRIPTION
**Pull request by FinalAngel**
* Added tests
* Cleaned up file structure
* Removed Django < 1.8 support
* Adapted ``README.txt``

### CODE REVIEW

File: aldryn_config.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# -*- coding: utf-8 -*-
+from aldryn_client import forms
```
Comment (czpython) [inline aldryn_config.py]: necessary?

File: djangocms_file/migrations/0003_remove_related_name_for_cmsplugin_ptr.py [hunk 2]
```diff
@@ -1,5 +1,4 @@
 # -*- coding: utf-8 -*-
-# Generated by Django 1.9.2 on 2016-02-26 14:00
```
Comment (czpython) [inline djangocms_file/migrations/0003_remove_related_name_for_cmsplugin_ptr.py]: this is added automatically by django, after 1.9 all migrations will have it :D

File: djangocms_file/models.py [hunk 3]
```diff
@@ -1,103 +1,188 @@
...
+    """
+    search_fields = ('name',)
+
+    TEMPLATE_CHOICES = [
+        ('default', _('Default')),
+    ]
+
+    template = models.CharField(
+        verbose_name=_('Template'),
+        choices=TEMPLATE_CHOICES + get_templates(),
```
Comment (czpython) [inline djangocms_file/models.py]: same feedback as djangocms-picture about template

File: aldryn_config.py (hunk 1 above)
Comment (FinalAngel) [inline aldryn_config.py]: yes by aldryn

Comment (czpython) [inline aldryn_config.py]: apps can have no config

**Comment by czpython** [comment]
👍

**Pull request merged**